db/migrate/20241101144625_remove_system_purpose_addons.rb

```diff
@@ -0,0 +1,35 @@
+class RemoveSystemPurposeAddons < ActiveRecord::Migration[6.1]
+  def up
+    remove_foreign_key :katello_subscription_facet_purpose_addons, :katello_purpose_addons, column: :purpose_addon_id
+
+    drop_table :katello_activation_key_purpose_addons
+    drop_table :katello_purpose_addons
+    drop_table :katello_subscription_facet_purpose_addons
```

Comment:
@parthaa Is this going to create problems with https://github.com/chris1984/katello/blob/sysporpose/db/migrate/20190605014649_add_purpose_addons.rb ?

do we need to do the "FakePurposeAddon" thing in that old migration file?